react/components/MinicartIconButton.tsx

```diff
@@ -14,20 +14,45 @@ interface Props {
   itemCountMode: MinicartTotalItemsType
 }
 
-const totalItemsSum = (arr: OrderFormItem[]) =>
-  arr.reduce((sum: number, product: OrderFormItem) => sum + product.quantity, 0)
+const countModeHandle = (
```

Comment:
This is more of a nitpick, but how about naming this `countItems` or `countCartItems`? Just because we tend to use `handlers` in event handlers

Reply:
@victorhmp Its done ;)